dashboard/config/locales/long_instructions.es-MX.yml

```diff
@@ -2111,8 +2112,8 @@ es-MX:
 
         Echa un ojo a un ejemplo de un manual
       OPD-K5 FirstProgram: Mira el siguiente video, que explica el primer programa
-        que los estudiantes más jóvenes escribirán en [Course A](https://studio.code.org/s/coursea-2018?section_id=2042045).
-        ¡En la página siguiente podrás escribir un programa similar!
+        que los estudiantes más jóvenes escribirán en [Course A][0]. ¡En la página
```

Comment:
oh, looks like the restoration failed here. The string looks correct in crowdin, though: https://crowdin.com/translate/codeorg/551/enus-esmx#q=OPD-K5+FirstProgram

Did something funky happen with the sync out?

Reply:
*restoration failed not just here, but everywhere in this file.

Reply:
@Hamms It seems something went wrong with the sync out.  Will it be better to re-run a sync down again before running the sync out again?

Reply:
I'd just re-run the sync out. I'm not sure what would cause redaction to work on most but not all files though...